Python/thread_pthread.h

```diff
@@ -359,6 +359,14 @@ PyThread_exit_thread(void)
     pthread_exit(0);
 }
 
+void _Py_NO_RETURN
+PyThread_hang_thread(void)
+{
+    while (1) {
+        pause();
```

Comment:
I'm afraid that it can change signals are received by other threads. I don't think that such zombie threading eating signals is a good idea.